src/components/AccountDropdown/AccountChildOption.tsx

```diff
@@ -16,18 +18,27 @@ type AccountChildRowProps = {
   cryptoBalance: string
   symbol: string
   onOptionClick: (accountId: AccountId) => void
+  chainId: ChainId
```

Comment:
@NeOMakinG do we need this? I.e `fromAccountId(accountId).chainId` will yield the `ChainId`  (or alternatively, no need to explicitly check for `isUtxoChainId()`, you can even use `isUtxoAccountId()`)

Reply:
IMO we don'tm I used `fromAccountId` inside another place and it's 100% the right thing to do, changed in [5143ec3](https://github.com/shapeshift/web/pull/7663/commits/5143ec32b38ab37e8d2a9ad4e93d2838deaa9c01)